### PR TITLE
Update 7-params.md

### PR DESCRIPTION
**Pull request by holtjohnson**
Cleaned up markdown that wasn't displaying clearly.

### CODE REVIEW

**Comment by bobcatfish** [comment]
hey thanks @holtjohnson !! We didn't notice this at all >.<

File: steps/7-params.md [hunk 1]
```diff
@@ -56,11 +56,10 @@ and that you are in your virtualenv.
    the tests that require them.
 
    Think of all the testing possibilities!
-  ```
+  ```bash
   
 4. When the tests run successfully, push them to your pull request:
 
-  ```bash
```
Comment (bobcatfish) [inline steps/7-params.md]: i think to fix this, you actually want to keep this line, and complete remove line 59 (the one you changed to `\`\`\`bash`).  If you preview your change by [looking at your fork](https://github.com/holtjohnson/catinabox/blob/master/steps/7-params.md) you can see that the number 4. is in the quote block now too.

I'm happy to make this change tho if you don't get a chance, thanks for bringing it to our attention!!

**Comment by holtjohnson** [comment]
@bobcatfish: I believe I've made the changes you asked for.

**Comment by bobcatfish** [comment]
Looks great @holtjohnson thanks very much :D

**Pull request merged**